src/promptflow-tracing/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# promptflow-tracing package
```

Comment:
change package -> changelog?
or just remove package?

Reply:
This is to align with the Python Library Reference.